packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -47,6 +47,7 @@ interface SchedulerCapableNetworkConstants {
   tokenList: TokenList;
   scheduledPaymentFeeFixedUSD: number;
   scheduledPaymentFeePercentage: number;
+  uniswapPairInitCodeHash: string;
```

Comment:
Can you add a comment in the code explaining why we need this. I think it would be good to explain this special case and mention the patch

Reply:
Comment added